### ISSUE TITLE
cpssd.net emails proposal

### ISSUE THREAD

**Issue by VoyTechnology** [iandioch/cpssd.net#28]
It would be pretty nice if we all got @cpssd.net emails, as well as group project emails (eg. paranoid@cpssd.net, feedlark@cpssd.net etc).

Since obviously nobody expects GMail like reliability, this can be done via self hosted https://github.com/flashmob/go-guerrilla, which will then just forward the emails to our own inboxes.

Is anybody interested in such a thing? I will be willing to do the work and host it if I can get MX records etc set up for cpssd.net

**Comment by byxor** [comment]
This would be nice, but I already have too many email addresses that I use for various things.

**Comment by iandioch** [comment]
I did think about this way back, but it was v. low priority.

I have used Mailgun before for @mail.safer.place emails. This seems to be a much better option than self-hosting an email service. It also allows sending as well as receiving through that address IIRC. There are other equivalent services, and I believe we'd be well within their free tiers. Maybe there is a limit on the number of accounts, I don't remember.

Right now CPSSD.net doesn't need us to manage any server, it's hosted on gh-pages, and I like it like that; there's practically zero maintenance required. I'd much prefer getting some email-aaS with the same ease of use.

I won't have time in the coming days, but maybe next week, to research the email providers and find a nice package for us. @VoyTechnology, do you want to do it this week? If we don't find a suitable one easily, we can self-host, sure.

**Comment by sentriz** [comment]
+1 for mailgun

**Comment by sentriz** [comment]
yep they do, it's because outgoing emails go to the recipients spam for some reason
i'll change it some other time 

I can only see a limit on domains, not accounts
should be fine, since it's only the one domain
